ci/owncloud/owncloud_options.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We definitly don't need more files for getting the credentials. Please allow only `~/.netrc`.

Reply:
In your comment on #1185 you mentioned having an ability to support getting credentials from an external file.
By default the script uses `.netrc` credentials, but this leaves the ability to easily import credentials without digging through configurations.